placesautocomplete/src/main/java/com/seatgeek/placesautocomplete/PlacesApi.java

```diff
@@ -160,7 +162,8 @@ public PlacesDetailsResponse details(final String placeId) throws IOException {
                 .appendPath(PATH_DETAILS)
                 .appendPath(PATH_JSON)
                 .appendQueryParameter(PARAMETER_KEY, googleApiKey)
-                .appendQueryParameter(PARAMETER_PLACE_ID, placeId);
+                .appendQueryParameter(PARAMETER_PLACE_ID, placeId)
+                .appendQueryParameter(PARAMETER_LANGUAGE, Locale.getDefault().toString());
```

Comment:
I don't believe that this is the correct value for this parameter. From reading the [documentation](https://developers.google.com/maps/faq#languagesupport) regarding the supported languages by the API, it would appear that the more appropriate value would be `Locale.getDefault().getLocation()`.

One more thing that I believe needs to be considered here is that we'll also likely want to add this parameter to the request in `autocomplete` [here](https://github.com/seatgeek/android-PlacesAutocompleteTextView/pull/17/files#diff-6a9211fabb94d0a6efdaec1583de97f8R134) for language consistency. It'd be a pretty odd experience to get autocomplete results in one language and then details in another.